hbase_sql/src/main/java/com/splicemachine/hbase/SpliceHFileCleaner.java

```diff
@@ -0,0 +1,51 @@
+package com.splicemachine.hbase;
+
+import com.splicemachine.EngineDriver;
+import com.splicemachine.access.HConfiguration;
+import com.splicemachine.backup.BackupManager;
+import com.splicemachine.backup.BackupUtils;
+import com.splicemachine.db.iapi.error.StandardException;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hbase.master.cleaner.BaseHFileCleanerDelegate;
+import org.apache.hadoop.hbase.util.FSUtils;
+import org.apache.log4j.Logger;
+
+/**
+ * Created by jyuan on 1/29/15.
+ */
+public class SpliceHFileCleaner extends BaseHFileCleanerDelegate {
```

Comment:
This one I think can be moved and config changed in spliceengine-ee.  We have to be able to modify/override config there anyway.